### ISSUE TITLE
Can't chose the option node (instead of browser) in eslint --init

### ISSUE THREAD

**Issue by Leonardo1807** [eslint/eslint#15245]
### Environment

Node version: v14.18.1
npm version: v8.1.2
Local ESLint version: v8.1.0 (Currently used)
Global ESLint version: Not found
Operating System: linux 5.14.14-artix1-1

### What parser are you using?

Default (Espree)

### What did you do?

I just executed `npx eslint --init` with the following options:
```
How would you like to use ESLint? … 
  To check syntax only
▸ To check syntax and find problems
  To check syntax, find problems, and enforce code style

What type of modules does your project use? … 
  JavaScript modules (import/export)
 ▸ CommonJS (require/exports)
  None of these
  
  Which framework does your project use? … 
  React
  Vue.js
 ▸ None of these
 
 Does your project use TypeScript? ‣ No

Where does your code run? …  (Press <space> to select, <a> to toggle all, <i> to invert selection)
✔ Browser
✔ Node <=

What format do you want your config file to be in? … 
  JavaScript
  YAML
▸ JSON
```
But no matter what I do, there is always this option in the `.eslintrc.json` file:
```json
{
    "env": {
        "browser": true,  // right here should be something like "node"
        [...]
    },
    [...]
}
```
It also says, that, after I finish the prompts, no local ESLint version was detected, although these both commands work:
```
$ npx eslint -v
8.1.0

$ ./node_modules/.bin/eslint -v
8.1.0
```
What can I do to prevent this?

### What did you expect to happen?

Node gets accepted and inserted in the `.eslintrc.json` file. My local ESLint installation should alsp be detected.

### What actually happened?

```
Local ESLint installation not found.
The config that you've selected requires the following dependencies:

eslint@latest
✔ Would you like to install them now with npm? · Yes
Installing eslint@latest
```

### Participation

- [X] I am willing to submit a pull request for this issue.

**Comment by snitin315** [comment]
@Leonardo1807 `browser` is selected by default. If you want to select only `node`, you must unselect the `browser` option by pressing space. And then navigate down to select `node` option by pressing `space` again.

**Comment by snitin315** [comment]
Here is a demo: 

https://user-images.githubusercontent.com/46647141/139970173-e4d635ed-d8ce-4c90-a1a9-c2e3579daa20.mov

**Comment by Leonardo1807** [comment]
> @Leonardo1807 `browser` is selected by default. If you want to select only `node`, you must unselect the `browser` option by pressing space. And then navigate down to select `node` option by pressing `space` again.
> 
> Or alternatively, press `i`, it will invert the selection from `browser` to `node`.

Sorry, I forgot to press space to remove browser. Now everything works fine 👍🏼
